### DISCUSSION TITLE
Creality CP-01

### DISCUSSION THREAD

**Discussion by CyberLele** [Desuuuu/DGUS-reloaded-Klipper#29] [Ideas]
Hello, I have a creality CP-01 where I installed your version of klipper and DGUS, I have a simple question: would it be possible to add the command G92 in the motion and homing screen to be able to make the origins for cnc and laser? Exactly as in the Creality firmware. Because now I have to give the command from the terminal, it would be convenient to do it directly from the touch screen! Thank you so much for the work you do! 
CyberLele.

**Comment by CyberLele** [comment]
https://user-images.githubusercontent.com/20773533/162272408-c6d90cda-824c-4a1a-9237-7a6e96c1e91b.mp4

**Comment by Desuuuu** [comment]
I don't plan on supporting this firmware anymore.

However you can try the [newer version](https://github.com/Desuuuu/klipper/discussions/74), which has support for custom menus, if you'd like.
